### DISCUSSION TITLE
It's choose, not chose in the new package description post install

### DISCUSSION THREAD

**Discussion by BeenAwhile** [vkbo/novelWriter#1329] [General]
Spell checker doesn't catch things like this.  I do.  It's a writing app, do it write.

**Comment by vkbo** [comment]
Typos happen. I assume you mean in the release notes? I'll fix it in time for the next patch. These notes are replaced for every minor release and sure, there are typos in them from time to time.

**Reply by LegolasMirkwood** [reply]
;)  I like the work you do.  When everybody helps a tiny little bit, it only gets better.  I'll get out of your way now.
